core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedServiceFactory.java

```diff
@@ -168,6 +168,11 @@
                     .put(Trace.class, HttpMethod.TRACE)
                     .build();
 
+    /**
+     * A value to specify if HTTP method annotation value is empty or not specified.
+     */
+    private static final String EMPTY_PATH = "";
```

Comment:
This is used in only one place. How about embedding it and leave a comment?

Reply:
I removed that constant. And I assigned the values myself and added a separate comment. 4508c1a